packages/components/src/DateTimePickers/shared/utils/test/dateMocking.js

```diff
@@ -1,26 +1,83 @@
 let OriginalDate;
```

Comment:
Yes it's actually clearly the wrong spot ^^'.
I also cross referenced it even harder across package in the form :D.
Maybe this could be moved in an other (new?) package or something like that with unit tests ?

Reply:
❓